07week/dodgeBall/dodgeBall.js

```diff
@@ -0,0 +1,175 @@
+const arrOfPeople = [
```

Comment:
Missing the test plane or tests,
Also missing the readme, code plan or comments to show your thought process, steps or assumptions.